### PR TITLE
Use simpler example for doctests and exceptions

### PR DESCRIPTION
**Pull request by wojtekmach**

### CODE REVIEW

File: lib/ex_unit/lib/ex_unit/doc_test.ex [hunk 1]
```diff
@@ -130,8 +130,8 @@ defmodule ExUnit.DocTest do
 
   You can also showcase expressions raising an exception, for example:
 
-      iex(1)> String.to_atom((fn -> 1 end).())
-      ** (ArgumentError) argument error
+      iex(1)> raise "some error"
+      ** (RuntimeErroro) some error
```
Comment (josevalim) [inline lib/ex_unit/lib/ex_unit/doc_test.ex]: Make sure this won't emit warnings when the suite is running. :)

**Comment by wojtekmach** [comment]
Eh, of course this was sent prematurely as I was making sure everything works. Should be fine now!

**Comment by wojtekmach** [comment]
Btw, we unfortunately don’t have `doctest ExUnit.DocTest` :’-( as we’re showing examples where it doesn’t work.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**